Content.Server/Zombies/PendingZombieComponent.cs

```diff
@@ -14,16 +14,51 @@ public sealed class PendingZombieComponent : Component
     {
         DamageDict = new ()
         {
-            { "Blunt", 0.5 },
-            { "Cellular", 0.2 },
+            { "Blunt", 0.8 },
             { "Toxin", 0.2 },
         }
     };
 
     [DataField("nextTick", customTypeSerializer:typeof(TimeOffsetSerializer))]
     public TimeSpan NextTick;
 
-    // Scales damage over time.
-    [DataField("infectedSecs")]
+    /// <summary>
+    /// Scales damage over time.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite), DataField("infectedSecs")]
     public int InfectedSecs;
+
+    /// <summary>
+    /// Number of seconds that a typical infection will last before the player is totally overwhelmed with damage and
+    ///   dies.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite), DataField("infectionLength")]
+    public float InfectionLength = 120f;
+
+    /// <summary>
+    /// Once the player enters crit, we ramp up the damage to ensure they don't suffer too long.
+    /// </summary>
+    [DataField("inCrit")]
+    public bool InCrit;
+
```

Comment:
this shouldnt be a datafield, also there is a IsCritical method, so use that instead of storing this

Reply:
You're making me tear my hair out here. Use the MobState event, don't use the event!